samples/XCT.Sample/ViewModels/Behaviors/ProgressBarAnimationBehaviorViewModel.cs

```diff
@@ -1,37 +1,31 @@
-﻿using System;
-using System.Threading.Tasks;
-using System.Windows.Input;
-using Xamarin.CommunityToolkit.ObjectModel;
+﻿using System.Windows.Input;
+using Xamarin.Forms;
 
 namespace Xamarin.CommunityToolkit.Sample.ViewModels.Behaviors
 {
 	public class ProgressBarAnimationBehaviorViewModel : BaseViewModel
 	{
 		double progress;
-		ICommand setTo0Command;
-		ICommand setTo50Command;
-		ICommand setTo100Command;
 
-		public double Progress
+		public ProgressBarAnimationBehaviorViewModel()
 		{
-			get => progress;
-			set
-			{
-				progress = value;
-				OnPropertyChanged();
-			}
+			SetTo0Command = new Command(() => SetProgress(0));
+			SetTo50Command = new Command(() => SetProgress(0.5));
+			SetTo100Command = new Command(() => SetProgress(1));
```

Comment:
For sure it was a low-value performance optimization, but we lost lazy initialization for these commands.
We will create them even if don't use.

Reply:
Yes and no. You're correct that we lose the lazy initialization, however, because `ProgressBarAnimationBehaviorPage` binds to each command, the binding process initiates the initialization of the Command, even when it implemented lazy loading:

```csharp
<Button Text="Set to 0" Command="{Binding SetTo0Command}"  />
<Button Text="Set to 50" Command="{Binding SetTo50Command}" />
<Button Text="Set to 100" Command="{Binding SetTo100Command}" />
```

In other words, if you put a breakpoint on `public ICommand SetTo0Command => setTo0Command ??= new Command(() => SetProgress(0));`, you will see it trigger before the button is tapped, because it first triggers when the bindings are initialized.